internal/parser/planparserv2/parser_visitor.go

```diff
@@ -383,18 +383,23 @@ func (v *ParserVisitor) VisitRelational(ctx *parser.RelationalContext) interface
 	leftValue, rightValue := getGenericValue(left), getGenericValue(right)
 
 	if leftValue != nil && rightValue != nil {
+		var ret *ExprWithType = nil
 		switch ctx.GetOp().GetTokenType() {
 		case parser.PlanParserLT:
-			return Less(leftValue, rightValue)
+			ret = Less(leftValue, rightValue)
 		case parser.PlanParserLE:
-			return LessEqual(leftValue, rightValue)
+			ret = LessEqual(leftValue, rightValue)
 		case parser.PlanParserGT:
-			return Greater(leftValue, rightValue)
+			ret = Greater(leftValue, rightValue)
 		case parser.PlanParserGE:
-			return GreaterEqual(leftValue, rightValue)
+			ret = GreaterEqual(leftValue, rightValue)
 		default:
```

Comment:
what about  EQ and NEQ

Reply:
yeah, u r right. thx for the reminder, and i'll fix it.